assets/xml/objects/object_sdn.xml

```diff
@@ -1,66 +1,66 @@
-        <Texture Name="object_sdn_Tex_00CFC0" OutName="tex_00CFC0" Format="ci8" Width="32" Height="32" Offset="0xCFC0" />
-        <Texture Name="object_sdn_Tex_00D3C0" OutName="tex_00D3C0" Format="i8" Width="16" Height="8" Offset="0xD3C0" />
-        <Texture Name="object_sdn_Tex_00D440" OutName="tex_00D440" Format="ci8" Width="16" Height="16" Offset="0xD440" />
-        <Limb Name="object_sdn_Standardlimb_00D540" Type="Standard" Offset="0xD540" />
-        <Limb Name="object_sdn_Standardlimb_00D54C" Type="Standard" Offset="0xD54C" />
-        <Limb Name="object_sdn_Standardlimb_00D558" Type="Standard" Offset="0xD558" />
-        <Limb Name="object_sdn_Standardlimb_00D564" Type="Standard" Offset="0xD564" />
-        <Limb Name="object_sdn_Standardlimb_00D570" Type="Standard" Offset="0xD570" />
-        <Limb Name="object_sdn_Standardlimb_00D57C" Type="Standard" Offset="0xD57C" />
-        <Limb Name="object_sdn_Standardlimb_00D588" Type="Standard" Offset="0xD588" />
-        <Limb Name="object_sdn_Standardlimb_00D594" Type="Standard" Offset="0xD594" />
-        <Limb Name="object_sdn_Standardlimb_00D5A0" Type="Standard" Offset="0xD5A0" />
-        <Limb Name="object_sdn_Standardlimb_00D5AC" Type="Standard" Offset="0xD5AC" />
-        <Limb Name="object_sdn_Standardlimb_00D5B8" Type="Standard" Offset="0xD5B8" />
-        <Limb Name="object_sdn_Standardlimb_00D5C4" Type="Standard" Offset="0xD5C4" />
-        <Limb Name="object_sdn_Standardlimb_00D5D0" Type="Standard" Offset="0xD5D0" />
-        <Limb Name="object_sdn_Standardlimb_00D5DC" Type="Standard" Offset="0xD5DC" />
-        <Limb Name="object_sdn_Standardlimb_00D5E8" Type="Standard" Offset="0xD5E8" />
-        <Limb Name="object_sdn_Standardlimb_00D5F4" Type="Standard" Offset="0xD5F4" />
-        <Skeleton Name="object_sdn_Skel_00D640" Type="Flex" LimbType="Standard" Offset="0xD640" />
-        <Animation Name="object_sdn_Anim_00DC7C" Offset="0xDC7C" />
+        <DList Name="gSoldierBottleDL" Offset="0x630" />
+        <DList Name="gSoldierBottleContentsDL" Offset="0x768" />
+        <DList Name="gSoldierBottleEmptyContentsDL" Offset="0x800" />
+        <Texture Name="gSoldierBottleTex" OutName="soldier_bottle_tex" Format="i4" Width="16" Height="16" Offset="0x808" />
+        <Animation Name="gSoldierStandWithHandOnChest" Offset="0xA54" />
+        <Animation Name="gSoldierDrink" Offset="0x20D8" />
+        <Animation Name="gSoldierCheerWithSpear" Offset="0x2A84" />
+        <Animation Name="gSoldierSitAndReach" Offset="0x3380" />
+        <Animation Name="gSoldierWave" Offset="0x3BFC" />
+        <Animation Name="gSoldierStandUp" Offset="0x4770" />
+        <Animation Name="gSoldierLookDown" Offset="0x4AC0" />
+        <Animation Name="gSoldierComeUpHere" Offset="0x5320" />
+        <Animation Name="gSoldierHalt" Offset="0x57BC" />
+        <Animation Name="gSoldierHalt2" Offset="0x5D28" />
+        <Animation Name="gSoldierHalt3" Offset="0x64C0" />
```

Comment:
I tried to name the animation based on what it shows in z64 utils, but the purpose of these animations wasn't clear. I'll ask darkeye but based on the comment at the top of the c file for this actor, this is likely an unused asset so it might not be there. Do we have any other way to help determine something like this?

Reply:
Might it be used by other actors that use this object? It's generally only better to document the used parts of an object unless you're working on the last actor to use it. Maybe StopHeishi?

Reply:
I checked with darkeye and in MM3D, they're named tome_03, tome_04 and tome_05. I guess tome means to stop/forbid/halt, but the numbers are different. Do you think I should keep what I have or should I match the numbers in MM3D?

Reply:
I think I'd rather they remain unnamed until we know where they are actually used.

Reply:
I agree with leaving them unnamed until StopHeishi is decompiled. Perhaps you could put a comment next to the animation explaining it, though. That way, your documentation doesn't go to waste.

In regards to the number situation, we don't have to follow the original names at all (otherwise, we would just use them when available), so definitely keep what you have. There's also the possibility that Grezzo renamed things or rearranged them in MM3D, so we shouldn't rely on the names too much.

Reply:
Okay, sure. So, you just mean revert these animations back to:
```
<Animation Name="object_sdn_Anim_0057BC" Offset="0x57BC" />
<Animation Name="object_sdn_Anim_005D28" Offset="0x5D28" />
<Animation Name="object_sdn_Anim_0064C0" Offset="0x64C0" />
```

Reply:
In terms of the *original* comment about naming them descriptively, it looks like tome03 is the start of his animation "halt" animation, tome04 is the animation of him shuffling side-to-side while blocking you, and tome05 is the idle loop of his "halt" animation. So maybe `gSoldierHaltStart`, `gSoldierHaltSidle` and `gSoldierHaltLoop`?

Reply:
> Okay, sure. So do you mean just revert these animations back to:
> 
> ```
> <Animation Name="object_sdn_Anim_0057BC" Offset="0x57BC" />
> <Animation Name="object_sdn_Anim_005D28" Offset="0x5D28" />
> <Animation Name="object_sdn_Anim_0064C0" Offset="0x64C0" />
> ```

Yep, that works. Definitely put a comment next to them with what you found, though; would be a shame to lose that documentation.

Reply:
Okay awesome, thanks everyone! I added the documentation in a comment beside each animation and reverted them back to their original names for now.